ts/store/middlewares/analytics.ts

```diff
@@ -445,6 +446,8 @@ const trackAction = (mp: NonNullable<typeof mixpanel>) => (
         choice: action.payload.choice,
         reason: action.payload.error.message
       });
+    case getType(sendAddCobadgeMessage):
+      return mp.track(action.type, { choice: action.payload });
```

Comment:
is it really a choice?
```suggestion
      return mp.track(action.type, { canAdd: action.payload });
```

btw what do you think to make the `sendAddCobadgeMessage` an action without payload?
At the end we want to know only these profiles that match this condition. These ones that never sent this action means `choice: false` implicitly

Reply:
In order to implement the required feature we need the payload, to distinguish and avoid sending the message to those who add a BANCOMAT and then shortly after also a cobadge card.
eg:
1) User add a BANCOMAT candidate to the addition of a cobadge card, we send the action `canAdd: true`.
2) The user add a co-badge card.
3) We send the notification message

if in step two we do not update the user's information, we will send the message even if the user has already inserted the card